Makefile

```diff
@@ -339,7 +339,8 @@ integration-functional-only: out/minikube$(IS_EXE) ## Trigger only functioanl te
 
 .PHONY: html_report
 html_report: ## Generate HTML  report out of the last ran integration test logs.
-	@go tool test2json -t < "./out/testout_$(COMMIT_SHORT).txt" > "./out/testout_$(COMMIT_SHORT).json"
+	@#TODO the json needs to be recorded when the go test is run, for timestamps to work properly
+	@go tool test2json < "./out/testout_$(COMMIT_SHORT).txt" > "./out/testout_$(COMMIT_SHORT).json"
```

Comment:
There was a side-effect of this, removing the timestamp also removes "Elapsed" (being parsed from the output)

```
{"Action":"run","Test":"TestFunctional/parallel/BuildImage"}
{"Action":"pause","Test":"TestFunctional/parallel/BuildImage"}
{"Action":"cont","Test":"TestFunctional/parallel/BuildImage"}
{"Action":"pass","Test":"TestFunctional/parallel/BuildImage"}
```

```
{"Time":"2021-05-03T08:00:09.027479789+02:00","Action":"run","Test":"TestFunctional/parallel/BuildImage"}
{"Time":"2021-05-03T08:00:09.027542553+02:00","Action":"pause","Test":"TestFunctional/parallel/BuildImage"}
{"Time":"2021-05-03T08:00:09.027553681+02:00","Action":"cont","Test":"TestFunctional/parallel/BuildImage"}
{"Time":"2021-05-03T08:00:09.027858066+02:00","Action":"pass","Test":"TestFunctional/parallel/BuildImage","Elapsed":0.69}

```